osx/Fuse/kbfuse.bundle/Contents/Extensions/10.11/kbfuse.kext/Contents/Info.plist

```diff
@@ -23,7 +23,7 @@
 		<string>MacOSX</string>
 	</array>
 	<key>CFBundleVersion</key>
-	<string>3.8.2</string>
+	<string>3.8.3</string>
 	<key>DTCompiler</key>
 	<string>com.apple.compilers.llvm.clang.1_0</string>
 	<key>DTPlatformBuild</key>
```

Comment:
is it expected all these bins disappeared? kbfuse/load_kbfsuse/mount_kbfuse?

Reply:
They didn't disappear, they just got a little smaller.  Not sure where github gets its percentages, but `kbfuse` for example just went from 146K to 137K.  Don't know why exactly, but doesn't seem like a big deal.